clip.cpp

```diff
@@ -895,6 +895,7 @@ std::string overzoom(const mvt_tile &tile, int oz, int ox, int oy, int nz, int n
 		static const std::string retain_points_multiplier_sequence = "tippecanoe:retain_points_multiplier_sequence";
 
 		for (auto feature : layer.features) {
+			feature.seq = 0;
```

Comment:
Curious why this is needed as it looks (to me) like seq is already zero initialized at https://github.com/felt/tippecanoe/blob/d9a5c2eed7a193850b7d1e9002b4ad68ec0d98b6/mvt.hpp#L58

Reply:
Hmm, you're right. I intentionally made it garbage to test, but it looks like it should already have been 0. Back to the drawing board!

Reply:
Okay, should this line then be removed or do you want to keep it for explicitness?

Reply:
I'll remove it. No need to intialize twice